code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -425,17 +427,18 @@
 	metabolization_rate = 2.5 * REAGENTS_METABOLISM
 	overdose_threshold = 125
 
-/datum/reagent/medicine/synthflesh/reaction_mob(mob/living/M, method=TOUCH, reac_volume,show_message = 1)
+/datum/reagent/medicine/synthflesh/reaction_mob(mob/living/M, method=TOUCH, reac_volume, show_message = 1, touch_protection, obj/item/bodypart/affecting)
```

Comment:
Undocumented/unrelated changes

Reply:
Related to patches caring about which body part you're targeting to do their effects.

Reply:
If you're worried about the new way of healing damage - it's the same thing but condensed into one line.